packages/projection/src/projectIntoSink.ts

```diff
@@ -121,32 +124,55 @@ export const projectIntoSink = <P extends object, PS extends P>(
 
   const source$ = combineLatest([props.sinks.buffer.tip$, props.sinks.buffer.tail$]).pipe(
     take(1),
-    mergeMap((blocks) =>
-      props
-        .chainSync({
-          points: blocksToPoints(blocks)
-        })
-        .pipe(
-          mergeMap((initialChainSync) => {
-            if (initialChainSync.intersection.point === 'origin') {
-              if (blocks[0] !== 'origin') {
-                throw new InvalidIntersectionError(
-                  "Couldn't find intersection within stability window buffer. Possible network configuration mismatch."
-                );
-              }
-              // buffer is empty, sync from origin
-              return syncFromIntersection(initialChainSync);
-            }
-            if (blocks[0] !== 'origin' && initialChainSync.intersection.point.hash !== blocks[0].header.hash) {
-              // rollback to intersection, then sync from intersection
-              return rollbackAndSyncFromIntersection(initialChainSync, blocks[1]);
+    mergeMap((blocks) => {
+      const points = blocksToPoints(blocks);
+      logger.info(`Starting projector with local tip at ${pointDescription(points[0])}`);
+
+      return props.cardanoNode.findIntersect(points).pipe(
+        take(1),
+        mergeMap((initialChainSync) => {
+          if (initialChainSync.intersection.point === 'origin') {
+            if (blocks[0] !== 'origin') {
+              throw new CardanoNodeErrors.CardanoClientErrors.IntersectionNotFoundError(
+                // TODO: CardanoClientErrors are currently coupled to ogmios types.
+                // This would be cleaner if errors were mapped to use our core objects.
+                points.map((point) =>
+                  point === 'origin'
+                    ? 'origin'
+                    : {
+                        hash: point.hash.toString(),
+                        slot: point.slot.valueOf()
+                      }
+                )
+              );
             }
-            // intersection is at tip$ - no rollback, just sync from intersection
+            // buffer is empty, sync from origin
             return syncFromIntersection(initialChainSync);
-          })
-        )
-    ),
-    withNetworkInfo(props.networkInfo)
+          }
+          if (blocks[0] !== 'origin' && initialChainSync.intersection.point.hash !== blocks[0].header.hash) {
+            // rollback to intersection, then sync from intersection
+            return rollbackAndSyncFromIntersection(initialChainSync, blocks[1]);
+          }
+          // intersection is at tip$ - no rollback, just sync from intersection
+          return syncFromIntersection(initialChainSync);
+        })
+      );
+    }),
+    // tap((evt) => {
```

Comment:
What's this?

Reply:
Trash, removed it in 9f54088fb256f79e8da2c838b1244e92618f25b2 (squashed to appropriate commit)